great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -48,11 +206,13 @@ def __init__(
         metric_name: str,
         metric_domain_kwargs: Optional[Union[str, dict]] = "$domain.domain_kwargs",
         metric_value_kwargs: Optional[Union[str, dict]] = None,
+        nan_raises_exception: Optional[Union[str, bool]] = False,
+        sampling_method: Optional[str] = "bootstrap",
         false_positive_rate: Optional[Union[float, str]] = 0.0,
-        round_decimals: Optional[Union[int, str]] = False,
         truncate_distribution: Optional[
             Union[Dict[str, Union[Optional[int], Optional[float]]], str]
         ] = None,
+        round_decimals: Optional[Union[int, str]] = False,
```

Comment:
I think the default value here should be changed to `None` (see also my other comment on `_get_round_decimals_using_heuristics`)
```suggestion
        round_decimals: Optional[Union[int, str]] = None,

Reply:
@anthonyburdi Great catch!  Will be fixed in the next commit.  Thanks for being so thorough!

Reply:
Done.